app/src/main/java/com/plaid/linksample/network/LinkSampleRetrofit.kt

```diff
@@ -0,0 +1,22 @@
+package com.plaid.linksample.network
+
+import okhttp3.OkHttpClient
+import retrofit2.Retrofit
+import retrofit2.adapter.rxjava2.RxJava2CallAdapterFactory
+import retrofit2.converter.gson.GsonConverterFactory
+
+class LinkSampleRetrofit {
```

Comment:
Would it make sense to make this a `LinkSampleApiFactory` and have it return the `LinkSampleApi` directly? 

- shifts Retrofit as being more of an implementation detail (not everyone may be using it)
- avoids accessing a `retrofit` property on a class that already has Retrofit in it's name
- allows to inject the `baseUrl` into the factory so we can provide all instructions on what parameters to change/tweak in the `Activity`